bot/monkey_patches.py

```diff
@@ -1,3 +1,5 @@
+import typing as t
+import unittest.mock
```

Comment:
uh, why exactly are we using unittest in production?

Reply:
I suppose I'm asking, is there a way to do what we need to do without mocking it?

Reply:
It's simpler than the alternative, but mocking in the main application is not necessarily a bad thing if used properly, See my other comments on this.